src/relentless/model/potential/angle.py

```diff
@@ -61,9 +61,17 @@ def derivative(self, type_, var, theta):
             is not a :class:`~relentless.variable.Variable`.
 
         """
+        # Validate theta
+        theta = self._validate_coordinate(theta)
 
```

Comment:
I noticed this is repeated in all of these children. I think you can just call `self._validate_coordinate` at the start of `BondedPotential.derivative`, and it will use the right one for the child class you're working with.
```suggestion
```